extensions/3rdparty/Verbatim/Verbatim.php

```diff
@@ -18,5 +18,14 @@ function wfVerbatimExtension( $parser ) {
 
 // The callback function for converting the input text to HTML output
 function renderVerbatim( $input ) {
+	global $wgEditInterfaceWhitelist;
+
+	// Begin wikia change
+	if (is_array($wgEditInterfaceWhitelist) and in_array(trim($input), $wgEditInterfaceWhitelist)) {
```

Comment:
Probably want to check `ucfirst` on the input as MW messages are case insensitive on the first letter.